lxd/db/exec/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package exec implements helpers around database/sql to execute various
+// kinds of frequently used SQL write statements.
+package exec
```

Comment:
The naming of this is rather unfortunate as it's conflicting with "os/exec" which we use quite widely through the code base. I don't expect we'll get any import clashes due to the db code being pretty self contained, but it's going to mess with my brain when I read through code reviews (as it just did when reviewing a later commit) and won't help us when we do the occasional grep through the code to see where we're using some low-level exec code.

Reply:
Sounds good to me, thanks.